.github/workflows/cd.yaml

```diff
@@ -0,0 +1,83 @@
+name: HOPR CD
+
```

Comment:
@jjperezaguinaga Could you document the purpose of the workflow here as well. So everyone touching it knows the context. I think most of the explanation is in the issues already.